server/test/required_data/config.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO server_informations VALUES ("config_duel_questions_per_round",5);
+INSERT INTO server_informations VALUES ("config_duel_rounds_per_duel",5);
+INSERT INTO server_informations VALUES ("config_question_timer_duration",10);
```

Comment:
I think you could add this in the sql update script, as they as supposed to be here when we launch the server :smile:

Reply:
done 👍